metrics/network/README.md

```diff
@@ -20,7 +20,10 @@ maximum number of (smallest sized) packets allowed through the transports.
 
 - `network-metrics-nuttcp.sh` measures the UDP bandwidth using nuttcp. This tool
 shows the speed of the data transfer for the UDP protocol.
- 
+
+- `network-latency.sh` measures the latency using ping. The ping utility measures
+how long it takes one packet to get from one point to another.
```

Comment:
@GabyCT Will be useful to add how the tests are run in the Readme, that is an example with the expected arguments for the script, so that one can get an idea without inspecting the script itself.